core/src/main/java/com/github/kaktushose/jda/commands/internal/register/TreeNode.java

```diff
@@ -45,7 +46,7 @@ public TreeNode(String name, SlashCommandDefinition command) {
     /// This guarantees to create a [CommandTree] that respects Subcommands and SubcommandGroups.
     public void addChild(String[] labels, SlashCommandDefinition command) {
         if (labels.length == 0) {
-            throw new IllegalArgumentException(
+            throw new JDACException.Internal(
                     "Failed to add child command: \"%s\". Cannot add child with empty labels! ".formatted(command.displayName()) +
                             "Please report this error to the devs of jda-commands."
```

Comment:
"Please report this error to the devs of jda-commands." can be removed, would be duplicated